.github/workflows/check-standalone-bayesian-optimization.yml

```diff
@@ -0,0 +1,31 @@
+name: Check instantiation of bayesian optimization on python 3.8
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install -r requirements-gpsearchers.txt --upgrade pip
```

Comment:
OK, I understand now, you are not installing requirements.txt.

Funny this works, the requirements do not even include numpy.

Reply:
AFAIK scipy depends on numpy so numpy would be installed together with that requirements